packet_analysis/equation/generate_equation.py

```diff
@@ -74,15 +74,15 @@ def get_segmented_average(conn: sqlite3.Connection, table: str, adjust_for_zero=
     return averaged_data, averaged_deviation
 
 
-# Seventh degree polynomial function
-def objective(x, a, b, c, d, e, f, g, h):
+# Fifth degree polynomial function
```

Comment:
Why did we change from to a 5th degree polynomial from 7th? Will this use less CPU when doing the complex calculations?

Reply:
Smaller degree, larger coefficients. The coefficient rounding was causing issues, and the degree of the polynomial was also more complex than necessary (Coefficients for the 6th and 7th expressions were minuscule)

Reply:
That makes sense. Thank you for clarifying.